.changeset/neat-lights-worry.md

```diff
@@ -1,13 +1,14 @@
 ---
+"@fluidframework/datastore-definitions": minor
 "@fluidframework/runtime-definitions": minor
 "@fluidframework/test-runtime-utils": minor
 ---
 ---
 "section": deprecation
 ---
 
-The function `process` on `IFluidDataStoreChannel` and `MockFluidDataStoreRuntime` is now deprecated.
+The function `process` on `IFluidDataStoreChannel`, `IDeltaHandler`, `MockFluidDataStoreRuntime` and `MockDeltaConnection` is now deprecated.
```

Comment:
Would be good on the other changeset as well, I see it kept the trailing period on the heading
```suggestion
The function `process` on `IFluidDataStoreChannel`, `IDeltaHandler`, `MockFluidDataStoreRuntime` and `MockDeltaConnection` is now deprecated
```